### PR TITLE
Renamed Mysterion in code

### PR DESCRIPTION
**Pull request by ignasbernotas**

### CODE REVIEW

File: src/app/bug-reporting/environment/environment-collector.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import type { SerializedLogCaches } from '../../logging/log-cache-bundle'
 import type { RavenData } from '../bug-reporter-metrics'
 
 interface EnvironmentCollector {
-  getMysterionReleaseId (): string,
+  getMysteriumVpnReleaseId (): string,
```
Comment (donce) [inline src/app/bug-reporting/environment/environment-collector.js]: Do we need to hardcode the actual name in such places? Maybe just `getApplicationReleaseId` or `getReleaseId` is enough? That way we wouldn't couple code to application name.

File: src/app/messages.js [hunk 2]
```diff
@@ -20,10 +20,10 @@ export default {
   countryListIsEmpty: 'No countries available',
   locationNotSelected: 'Please select location',
   initializationError: {
-    message: 'Failed to load Mysterion'
+    message: 'Failed to load MysteriumVpn'
   },
-  processInstallationPermissionsError: 'Failed to install Mysterion. Please restart the app and grant permissions.',
-  processInstallationError: 'Failed to install Mysterion.',
+  processInstallationPermissionsError: 'Failed to install MysteriumVpn. Please restart the app and grant permissions.',
+  processInstallationError: 'Failed to install MysteriumVpn.',
```
Comment (donce) [inline src/app/messages.js]: Maybe `MysteriumVPN`?

File: src/app/logging/winston.js [hunk 3]
```diff
@@ -57,13 +57,13 @@ function createWinstonSyncComLogger (communication: SyncRendererCommunication) {
...
+function mapWinstonLogLevelToMysteriumVpnLevel (level: WinstonLevel): LogLevel {
   return level === 'error' ? 'error' : 'info'
 }
 
 export type { WinstonLogEntry }
 export {
   createWinstonCachingLogger,
   createWinstonSyncComLogger,
-  mapWinstonLogLevelToMysterionLevel
+  mapWinstonLogLevelToMysteriumVpnLevel
```
Comment (donce) [inline src/app/logging/winston.js]: We can also get rid of application name here, i.e. `mapWinstonLogLevelToApplicationLevel`

File: src/app/messages.js (hunk 2 above)
Comment (shroomist) [reply src/app/messages.js]: I think we're going with abbreviations uppercase in code

File: src/app/bug-reporting/environment/environment-collector.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ import type { SerializedLogCaches } from '../../logging/log-cache-bundle'
 import type { RavenData } from '../bug-reporter-metrics'
 
 interface EnvironmentCollector {
-  getMysterionReleaseId (): string,
+  getReleaseId (): string,
```
Comment (shroomist) [inline src/app/bug-reporting/environment/environment-collector.js]: how about`getMysteriumVPNAppReleaseID()` 🤣

File: src/app/bug-reporting/environment/main-environment-collector.js [hunk 5]
```diff
@@ -25,17 +25,17 @@ import { BugReporterMetrics } from '../bug-reporter-metrics'
...
   _logCacheBundle: LogCacheBundle
-  _mysterionReleaseId: string
+  _releaseId: string
   _bugReporterMetrics: BugReporterMetrics
 
-  constructor (logCacheBundle: LogCacheBundle, mysterionReleaseId: string, bugReporterMetrics: BugReporterMetrics) {
+  constructor (logCacheBundle: LogCacheBundle, releaseId: string, bugReporterMetrics: BugReporterMetrics) {
     this._logCacheBundle = logCacheBundle
-    this._mysterionReleaseId = mysterionReleaseId
+    this._releaseId = releaseId
```
Comment (shroomist) [inline src/app/bug-reporting/environment/main-environment-collector.js]: ?? `mysteriumVPNAppReleaseID`?

Comment (ignasbernotas) [reply src/app/bug-reporting/environment/main-environment-collector.js]: what for?

File: src/app/bug-reporting/environment/environment-collector.js (hunk 4 above)
Comment (ignasbernotas) [reply src/app/bug-reporting/environment/environment-collector.js]: why do we need to couple it with the app name?

File: test/unit/app/sync-callbacks-initializer.spec.js [hunk 6]
```diff
@@ -28,7 +28,7 @@ import type { RavenData } from '../../../src/app/bug-reporting/bug-reporter-metr
 import { TAGS } from '../../../src/app/bug-reporting/bug-reporter-metrics'
 
 class MockEnvironmentCollector implements EnvironmentCollector {
-  mockMysterionReleaseId = 'mock mysterion release id'
+  mockMysteriumVpnReleaseId = 'mock mysterium vpn release id'
```
Comment (donce) [inline test/unit/app/sync-callbacks-initializer.spec.js]: `mockReleaseId`

File: src/main/dependencies/modules/application.js [hunk 7]
```diff
@@ -18,23 +18,23 @@
...
-import { getMysterionReleaseId } from '../../../libraries/version'
+import { getReleaseId } from '../../../libraries/version'
 
 function bootstrap (container: Container) {
   const version = process.env.MYSTERION_VERSION
   const build = process.env.BUILD_NUMBER
-  const mysterionReleaseID = getMysterionReleaseId(version, build)
-  global.__mysterionReleaseID = mysterionReleaseID
+  const mysteriumVpnReleaseID = getReleaseId(version, build)
+  global.__mysteriumVpnReleaseID = mysteriumVpnReleaseID
```
Comment (donce) [inline src/main/dependencies/modules/application.js]: `global.__releaseId`

**Pull request merged**